test/system/comments_test.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+
+class CommentsTest < ApplicationSystemTestCase
+  def curiosity
+    @curiosity ||= create(:curiosity_card, content: 'simple content')
+  end
+
+  def user
+    @user ||= create(:user)
+  end
+
+  setup do
+    curiosity
+    user
+  end
+
+  test 'Creating a comment' do
```

Comment:
Your tests name should be clear on the specs 
A test `Creating a comment` says nothing but the initial scenario. I believe a better name could be something like `Logged in user can create a comment`, `A comment can be created`, `Success creating a comment`

Reply:
This one applies for the rest of your tests